traits/ctraits.c

```diff
@@ -2974,18 +2974,63 @@ trait_traverse(trait_object *trait, visitproc visit, void *arg)
 |  Handles the 'getattr' operation on a 'CTrait' instance:
 +----------------------------------------------------------------------------*/
```

Comment:
This comment is now out of place; it needs to be moved so  that it's back above the definition of `trait_get_attro`.